doc/how_to/custom_components/reactive_html/reactive_html_dataframe.md

```diff
@@ -0,0 +1,66 @@
+# DataFrames and ReactiveHTML
+
+In this guide you will learn how to efficiently implement `ReactiveHTML` components with a
+`DataFrame` parameter.
+
+## Creating a Table Pane
```

Comment:
Would it help users more if the example was based on Tabulator? That would enable users to implement advanced Tabulator features if needed.

Reply:
I'd say no, I don't think implementing your own Tabulator wrapper will ever be a good route to go.

Reply:
It would not be implementing everything. Instead it would be implementing a specific use case that Panels Tabular does not support. Maybe you need some special JavaScript event handling or images in the cells or ....

Reply:
As I see ReactiveHTML, it is both for reusable components and for one offs.